### ISSUE TITLE
add mlflow metric logging function

### ISSUE THREAD

**Issue by pascal456** [fhdw-forschung/modelling-tools#64]
# draft
```python
from sklearn.metrics import mean_absolute_error
from sklearn.metrics import mean_absolute_percentage_error
from sklearn.metrics import mean_squared_error
from sklearn.metrics import mean_squared_log_error
from sklearn.metrics import r2_score

def log_metrics_to_mlflow(y_pred, y_true):
    mean_absolute_error
    mean_absolute_percentage_error
    mean_squared_error
    mean_squared_log_error
    r2_score
```
# ideas 
- should return a `dict` so that it can directly utilize mlflow.log_metrics

# note

- could form a part of an evaluation module
- related to https://github.com/fhdw-forschung/modelling-tools/issues/53
